public_html/wp-content/mu-plugins/blocks/assets/src/shared/featured-image/inspector-control.js

```diff
@@ -2,39 +2,15 @@
  * WordPress dependencies.
  */
 const { Component } = wp.element;
-const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl, ToggleControl } = wp.components;
+const { PanelBody, PanelRow, TextControl, BaseControl, SelectControl } = wp.components;
 const { __ } = wp.i18n;
-import './style.scss';
 
 /**
- * Not sure if these sizes are actually useful.
+ * Internal dependencies.
  */
-const sizePresets = [
-	{
-		label : __( 'Thumbnail', 'wordcamporg' ),
-		value : '150',
-	},
-	{
-		label : __( 'Thumbnail - Large', 'wordcamporg' ),
-		value : '926',
-	},
-	{
-		label : __( 'Medium', 'wordcamporg' ),
-		value : '300',
-	},
-	{
-		label : __( 'Medium - Large', 'wordcamporg' ),
-		value : '768',
-	},
-	{
-		label : __( 'Large', 'wordcamporg' ),
-		value : '1024',
-	},
-	{
-		label : __( '(Custom)', 'wordcamporg' ),
-		value : '',
-	},
-];
+
+import './style.scss';
+import { AvatarSizeControl } from '../avatar/inspector-controls'
```

Comment:
```suggestion
import { AvatarSizeControl } from '../avatar'
```

[SHOULD] The index file in the avatar directory exports components from the other directory files as well, so this can be simplified.

Reply:
done